src/components/pages/CustomerDashboard/cust-tabs.js

```diff
@@ -25,12 +27,18 @@ const CustTab = () => {
   const { resultInfo } = useContext(FormContext);
   const { custInfo } = useContext(CustomersContext);
   const { getCustomerByID } = useContext(APIContext);
+  const { getPet } = useContext(APIContext);
+  // const { pets } = useContext(PetsContext); // testing that context is updated before pets are rendered in RenderPetProfile
 
   useEffect(() => {
     getCustomerByID(authState);
     // eslint-disable-next-line react-hooks/exhaustive-deps
+    getPet(authState);
   }, []);
```

Comment:
make sure to check if any of the dependencies need to be in there before we merge it into main.

Reply:
I was going to test this, but it might not be necessary now. For some reason, the dependency array warning has stopped appearing when I run the program. I don't understand what caused it, initially, but now its operating in the manner that I expected, using the empty array to cause the useEffect to run only once, on load.